### DISCUSSION TITLE
exporters follow up

### DISCUSSION THREAD

**Discussion by YousefED** [TypeCellOS/BlockNote#2487] [Ideas & Enhancements]
https://github.com/TypeCellOS/BlockNote/pull/1143 ships the first version of pdf / docx exporters. This issue collects possible improvements:

- code block
- fixed table widths for pdf instead of 100% width (difficult, would require a table layout algorithm)
- multi-level bullet list indentation styles (change from solid to open dot to square etc) 
- bug: support both strike and underline for same text
- support for audio / video files or preview images
- page break elements

**Comment by woutervroege** [comment]
Re-linking https://github.com/TypeCellOS/BlockNote/issues/2226

**Comment by Sylvenas** [comment]
Page breaks are one of the most urgent requirements when exporting.
